pkg/cmd/server/start/master_args.go

```diff
@@ -219,7 +230,13 @@ func (args MasterArgs) BuildSerializeableEtcdConfig() (*configapi.EtcdConfig, er
 		StorageDir:    args.EtcdDir,
 	}
 
+	if args.ListenArg.UseTLS() {
```

Comment:
from args, you get no options for etcd server addresses... it uses the scheme from --listen, the host from --master, and port 4001. I'll add a follow up item to split listen args for etcd and node if we think it's important